packages/contentstack-utilities/src/helpers.ts

```diff
@@ -8,3 +9,21 @@ export const doesBranchExist = async (stack, branchName) => {
       return error;
     });
 };
+
+export const isManagementTokenValid = async (stackAPIKey, managementToken) => {
+  const httpClient = new HttpClient({ headers: { api_key: stackAPIKey, authorization: managementToken } });
+  try {
+    const response = (await httpClient.get(`${configHandler.get('region').cma}/v3/environments?limit=1`))?.data;
+
+    if (response?.environments) {
+      return { valid: true, message: `valid token and stack api key` }
+    } else if(response?.error_code) {
+      return { valid: false, message: response.error_message };
+    } else {
+      throw typeof response === "string"? response : "";
+    }
+  } catch (error) {
+    cliux.print(`Failed to check the validity of the Management token. ${error}`, {color:"red"});
+    process.exit(1)
```

Comment:
Same comment mentioned above applies here as well. Did we covered the unit test cases for these lines?

Reply:
Removed the process.exit